tests/platforms/kubernetes/appmanager_test.go

```diff
@@ -363,20 +423,34 @@ func TestWaitUntilServiceStateAndGetExternalURL(t *testing.T) {
 		client := newFakeKubeClient()
 		// Set up reactor to fake verb
 		client.ClientSet.(*fake.Clientset).AddReactor(
-			"get",
+			getVerb,
 			"services",
 			func(action core.Action) (bool, runtime.Object, error) {
 				ns := action.GetNamespace()
 				assert.Equal(t, testNamespace, ns)
 
 				obj := &apiv1.Service{
 					Spec: apiv1.ServiceSpec{
-						ExternalIPs: []string{},
+						Ports: []apiv1.ServicePort{},
+					},
+					Status: apiv1.ServiceStatus{
+						LoadBalancer: apiv1.LoadBalancerStatus{
+							Ingress: []apiv1.LoadBalancerIngress{},
+						},
 					},
 				}
 
 				if getVerbCalled == 2 {
```

Comment:
Same magic number.

Reply:
I would add the comment rather than using constant.